vsintegration/tests/FSharp.Editor.Tests/Helpers/RoslynHelpers.fs

```diff
@@ -201,7 +207,27 @@ type TestHostServices() =
     inherit HostServices()
 
     override this.CreateWorkspaceServices(workspace) =
-        TestHostWorkspaceServices(this, workspace) :> HostWorkspaceServices
+        TestHostWorkspaceServices(this, workspace)
```

Comment:
I wonder if this is the right place to do this. It's a shared module and this function, while having quite a benign name, now overrides the defaults of some settings. I'd probably prefer having non-default settings explicitly specified per each test suite.

Reply:
Yes, this is provisional. I was mostly concerned with not losing the settings that were already different from defaults in when in tests. We can in fact now import EditorOptions at any point during testing and make changes.

Reply:
About the Shared attribute. I added it because I noticed running VS in debug that EditorOptions get constructed quite often. In tests this should not be shared, and if it is, we could restrict this with scopes. It's something to look into.

Reply:
So once again, why is this needed here? I'm under the impression this actually doesn't affect the test outcome now

Reply:
Agree, we could yeet this if it's not needed for current tests.